packages/mermaid/src/diagrams/class/classDiagram.spec.ts

```diff
@@ -264,6 +264,93 @@ class C13["With Città foreign language"]
       const str = 'classDiagram\n' + 'note "test"\n';
       parser.parse(str);
     });
+
+    const keywords = [
+      'direction',
+      'classDiagram',
+      'classDiagram-v2',
+      'namespace',
+      '}',
```

Comment:
Why only one bracket? Is there a special meaning to that? May be opening bracket is missing (as a separate line)

Reply:
This should've been two brackets. These specific brackets put the parser in a `namespace-body` state, so this test would ensure that 1. these brackets are allowed inside strings. 2. They throw an error if they appear after the `note` keyword without quotes.